CodeGeneration/Tests/ValidateSyntaxNodes/ValidateSyntaxNodes.swift

```diff
@@ -383,7 +383,7 @@ class ValidateSyntaxNodes: XCTestCase {
           node: .regexLiteralExpr,
           message: "child 'ClosingPounds' has a token as its only token choice and should thus be named 'ExtendedRegexDelimiter'"
         ),
-        ValidationFailure(node: .sourceFile, message: "child 'EOFToken' has a token as its only token choice and should thus be named 'EOF'"),
+        ValidationFailure(node: .sourceFile, message: "child 'EndOfFileToken' has a token as its only token choice and should thus be named 'EndOfFile'"),
```

Comment:
No, I think `EndOfFileToken` is a good name here. I have a comment in my renamed changes (haven’t opened a PR for it yet) that says the following: “We should explicitly mention token here because it’s not obvious that the end of a file is represented by a token”